html/admin/api/projects/crew/crewRoles/list.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once __DIR__ . '/../../../apiHeadSecure.php';
+
+if (!$AUTH->instancePermissionCheck(124)) die("404");
```

Comment:
is 404 the right response here? @Jbithell can correct me but I think the standard across the project is to use the "finish" function with an error message set

Reply:
AFAIK there's a mix of both, but generally 404s are returned when permission is the problem

Reply:
This follows the existing php file's action so I would keep it the same?